### PR TITLE
openldap: create -tools subpackage.

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: srcpkgs/openldap/template [hunk 1]
```diff
@@ -15,16 +15,17 @@ configure_args="--prefix=/usr
...
 license="OpenLDAP License v2.8 - BSD alike"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 homepage="http://www.openldap.org"
-distfiles="ftp://ftp.openldap.org/pub/OpenLDAP/openldap-release/openldap-$version.tgz"
+distfiles="ftp://ftp.openldap.org/pub/OpenLDAP/openldap-release/openldap-${version}.tgz"
 checksum=cdd6cffdebcd95161a73305ec13fc7a78e9707b46ca9f84fb897cd5626df3824
 
 system_accounts="ldap"
-ldap_homedir="/var/lib/openldap"
+ldap_homedir=/var/lib/openldap
```
Comment (Duncaen) [inline srcpkgs/openldap/template]: Why unquoting this?

Comment (maxice8) [reply srcpkgs/openldap/template]: it shouldn't be, my formalizing script should be ignoring fixed strings on username related variables like system_accounts and _homedir.

File: srcpkgs/openldap/template [hunk 2]
```diff
@@ -57,3 +58,27 @@ libldap_package() {
 		vmove "usr/lib/*.so.*"
 	}
 }
+
+openldap-tools_package() {
+	short_desc+=" - utilities"
+	pkg_install() {
```
Comment (Ypnose) [inline srcpkgs/openldap/template]: I think you forgot `vmove usr/bin/ldapadd`

**Comment by maxice8** [comment]
On Fri, Jun 22, 2018 at 05:12:16AM -0700, Ypnose wrote:
> Ypnose commented on this pull request.
> 
> 
> 
> > @@ -57,3 +58,27 @@ libldap_package() {
>  		vmove "usr/lib/*.so.*"
>  	}
>  }
> +
> +openldap-tools_package() {
> +	short_desc+=" - utilities"
> +	pkg_install() {
> 
> I think you forgot `vmove usr/bin/ldapadd`
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/307#pullrequestreview-131170486

Yes, i removed all symlinks from matching because they all point to /usr/libexec/ldap or something, turns out there is one that doesn't.

**Comment by Ypnose** [comment]
@Duncaen Do you know when it will be merged?

**Comment by maxice8** [comment]
On Tue, Jul 03, 2018 at 03:31:03PM +0000, Duncan Overbruck wrote:
> Duncaen approved this pull request.
> 
> Looks fine to me, @maxice8 can merge if its ready.
> 
> 
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/307#pullrequestreview-134052651

alright, once the fallout of ffmpeg is resolved i'll merge

**Pull request closed without merging**